Tenduke.Client.WPFSample/Tenduke.Client.WPFSample.csproj

```diff
@@ -52,12 +51,12 @@
     </None>
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Update="chromiumembeddedframework.runtime.win-arm64" Version="129.0.11" />
+    <PackageReference Include="chromiumembeddedframework.runtime.win-arm64" Version="135.0.17" />

```

Comment:
Do these need to be included in the sample programs? Is it possible to build and run with the embedded browser as a transitive dependency?

The [documentation for the library](https://github.com/10Duke/10duke-dotnet-client/tree/master/Tenduke.Client.WinForms#embedded-browser) implies that the dependencies are all included in the library and does not mention the client application needed to reference CEF directly.